userguide/advanceduse/reverse-tethering.rst

```diff
@@ -1,34 +1,76 @@
 Reverse tethering
 =================
 
-This tutorial helps you to connect your Ubuntu Touch device to your computer to access the Internet on it.
+This tutorial helps you to connect your Ubuntu Touch device to your computer to access the internet on it.
 This is useful if there is no available Wi-Fi connection or you don't have a data subscription on your Ubuntu Touch device.
```

Comment:
Honestly, after re-reading this now after a very long time I find it confusing what we're saying. Maybe it could be made clearer what is "it". Which device has internet by itself and which device is receiving internet now?!

Reply:
Tried to rephrase it. Maybe it's clearer now.